docs/hacking.md

```diff
@@ -35,4 +35,39 @@ To get the version of the images that are running within crane-operator workload
         - name: CRANE_SECRET_SERVICE_IMAGE
           value: <crane-secret-service-image>
     ```
-    **Note**: Omit any of the env variables(name/value) related to images that needs to be set as default from above. 
\ No newline at end of file
+    **Note**: Omit any of the env variables(name/value) related to images that needs to be set as default from above.
+4. To get commitID of the image that is being used run the following command - 
+    ```shell script
+    docker pull <image>
+    docker inspect <image> | grep commit
+    ```
+    For example, 
+    ```shell script
+    docker pull quay.io/konveyor/crane-operator-container:latest
+    latest: Pulling from konveyor/crane-operator-container
+    f70d60810c69: Already exists 
+    545277d80005: Already exists 
+    de516cc59493: Pull complete 
+    80a491f42542: Pull complete 
+    acd6680814d4: Pull complete 
+    Digest: sha256:53c7ed89a431e032dbeeb5069342b963a1467647753bec6f436b5ec0dbcdce7a
+    Status: Downloaded newer image for quay.io/konveyor/crane-operator-container:latest
+    quay.io/konveyor/crane-operator-container:latest
+    ```
+   And then,
+   ```shell script
+    docker inspect quay.io/konveyor/crane-operator-container:latest | grep commit
```

Comment:
@JaydipGabani , I'm not really sure the context for determining the commit, but cleaner way might be to use something like:
`[docker|podman] inspect --format 'commit {{index .Labels "io.openshift.build.commit.id"}} for {{index .RepoTags}}' <image>`

example:

```
podman inspect --format 'commit {{index .Labels "io.openshift.build.commit.id"}} for {{index .RepoTags}}' quay.io/konveyor/crane-operator-container:latest quay.io/konveyor/crane-secret-service:latest

commit 611ab30d96f76e6904f4f43ce7001b422749b431 for [quay.io/konveyor/crane-operator-container:latest]
commit 017d65b1eab3760d82649e666a76dd9e2ebcc145 for [quay.io/konveyor/crane-secret-service:latest]
```